onnx_tf/handlers/backend/sequence_at.py

```diff
@@ -0,0 +1,45 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("SequenceAt")
+class SequenceAt(BackendHandler):
+
+  @classmethod
+  def chk_pos_in_bounds(cls, input_seq, pos):
+    """
+    Check the position is in-bounds with respect to the sequence.
+    Accepted range for 'position' is in [-n, n - 1], where n is the
+    number of tensors in 'input_sequence'.
+
+    :param input_seq: input sequence
+    :param pos: position of the output tensor
+
+    :return: True if position is in-bounds or input length is dynamic.
+    """
+    seq_length = input_seq.shape[0].value
+
+    if seq_length is None: return True
+
+    seq_length = tf.cast(seq_length, tf.int32)
+    pos = tf.cast(pos, tf.int32)
```

Comment:
Cast into `int64` because the spec allows `position` to be `int32` or `int64`. Same comment for `sequence_erase.py` and `sequence_insert.py`.

Reply:
Good catch. I changed it to the pos.dtype instead of int64. Please review again. Thanks!